protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -3574,7 +3574,7 @@ message ReferralProgram {
   // Defined tiers in increasing order. First element will give Tier 1, second
   // element will give Tier 2, and so on.
   repeated vega.BenefitTier benefit_tiers = 3;
-  // Timestamp as Unix time in seconds, after which when the current epoch ends, the
+  // Timestamp as Unix time in nanoseconds, after which when the current epoch ends, the
   // programs status will become STATE_CLOSED and benefits will be disabled.
```

Comment:
No relevant to this PR but just noticed: it says that `the programs status will become STATE_CLOSED` when there isn't a "state" field on the API?